src/db/sysdb.h

```diff
@@ -804,6 +804,7 @@ struct sysdb_upgrade_ctx {
 
 int sysdb_init_ext(TALLOC_CTX *mem_ctx,
                    struct sss_domain_info *domains,
+                   bool dont_create_cache,
```

Comment:
Maybe use `create_cache` instead of the negation?

Reply:
Agree, should be more readable; will update.

Reply:
`create_missing_cache`

Reply:
ack

Reply:
Updated (+ rebased)

Reply:
And added one more patch to removed unused define.